sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/_quickpulse/_live_metrics.py

```diff
@@ -150,3 +165,21 @@ def _record_log_record(self, log_data: LogData) -> None:
                     exc_message = log_record.attributes.get(SpanAttributes.EXCEPTION_MESSAGE)
                     if exc_type is not None or exc_message is not None:
                         self._exception_rate_counter.add(1)
+
+
+# pylint: disable=unused-argument
+def _get_process_memory(options: CallbackOptions) -> Iterable[Observation]:
+    # rss is non-swapped physical memory a process has used
+    yield Observation(
+        PROCESS.memory_info().rss,
+        {},
+    )
+
+
+# pylint: disable=unused-argument
+def _get_processor_time(options: CallbackOptions) -> Iterable[Observation]:
+    # Processor time does not include idle time
+    yield Observation(
```

Comment:
If it's only a single value, what's the purpose of yielding instead of returning?

Reply:
Not much difference in this case. It's just consistent with our examples.